cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -634,3 +696,34 @@ def _revert():
 
     view = navigate_to(ansible_service, "Details")
     assert view.provisioning.results.get_text_of("Status") == "successful"
+
+
+@pytest.mark.tier(3)
+@pytest.mark.provider([VMwareProvider], override=True)
+@pytest.mark.usefixtures("setup_provider")
+def test_ansible_service_linked_vm(
+    appliance,
+    new_vm,
+    ansible_policy_linked_vm,
+    ansible_service_request,
+    ansible_service,
+    request,
+):
+    """Check Whether service has associated VM attached to it.
+
+    Bugzilla:
+        1448918
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        initialEstimate: 1/3h
+        tags: ansible_embed
+    """
+    added_tag = new_vm.add_tag()
+    request.addfinalizer(lambda: new_vm.remove_tag(added_tag))
```

Comment:
Since the VM is function scoped and is cleaned up after the test, is it worthwhile to spend the time removing the tag?

Reply:
My thought is whatever is added/created should be removed in `finalizer`, if you suggests to remove `remove_tag()` I will remove it.

Reply:
Removed Finalizer for `remove_tag()`